hr_attendance_report_theoretical_time/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,28 @@
+You need to be at least "Attendance / Manual Attendance" for being able to see
```

Comment:
At least you have to be Officer because [parent menu](https://github.com/odoo/odoo/blob/b08aa26d8fd453f94b41307ac442489cf71e949a/addons/hr_attendance/views/hr_attendance_view.xml#L187) requires it.

Reply:
This wasn't this way in v10, but on v12 this menu was accessible by all, so you should turn to the changes I was doing in v10 for this. There's a record rule for making that users with manual attendance only see their attendances any way.

Reply:
Still being an inconsistency between readme and module functionality at least in v11.
If you want to keep as this way please change readme

Reply:
Did you attend this comment?

Reply:
Sorry, i forgot, just changed it in #576